content/docs/atom-guide/look-dev/tools/material-editor.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Material Editor
+description: Learn how to use the Material Editor to create materials in Atom.
+toc: true
+---
+
+## Overview
+The **Material Editor** is a standalone application that allows artists to view, edit, and create new materials. This section introduces the Material Editor and describes the editor layout and its individual panels.
+
+For more information about features common to Atom tools, menus, dockable panels, and working with documents, see [Atom Tools Common Features](/docs/atom-guide/look-dev/tools/atom-tools-common-features/). 
+
+## Quick Start
+To get started, launch the Material Editor and create or edit a material. 
+
+### Launching Material Editor
+There are multiple ways to launch Material Editor.
+- From **Open 3D Engine (O3DE)** Editor select **Main Menu > Tools > Material Editor**
+- From the Material Component context menu, select **Open Material Editor...**
+- From the Asset Browser, double click on a ".material" or ".materialtype".
+- From the Asset Browser, right click on a ".material" or ".materialtype", then select **Open in Material Editor...** 
+- Material Editor is also a standalone executable that can be launched directly from a file browser or command console.
+  - Launch the executable `<build>\bin\profile\MaterialEditor.exe`
+  - This requires passing the �project-path, followed by the path your project, as a command line argument.
```

Comment:
That is correct as long as the project graph is a relative or absolute path from where the command was called. If it was from a file browser you would either have to open a command console, have a shortcut, or type it into the address bar to add the project path. There is also another way to set the project path in the settings registry so that you don't have to type it in. But that's not commonly advertised.